src/encounters/RufusEncounter.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable class-methods-use-this */
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import Encounter from './Encounter';
+import { addTreat } from '../reducers/gameReducer';
+
+
+class VeraEncounter extends Encounter {
+  getInitialText() {
+    return `Rufus is playing with a coin. "Say, wanna play a little game? Tails I win, heads you do. Let's wager... five treats."`;
+  }
+
+  getCharacter() {
+    return 'rufus';
+  }
+
+  getRelevantStats() {
+    const { treatCount } = this.props;
+    return [
+      {
+        name: 'Treats',
+        value: treatCount,
+      },
+    ];
+  }
+
+  getTitle() {
+    return 'Rufus Scippio';
+  }
+
+  getInteractions() {
+    const { treatCount } = this.props;
+    const treatGift = Math.random() > 0.5 ? 5 : -5;
+    return [
+      {
+        type: 'treat',
+        isAvailable: treatCount >= 5,
+        buttonText: `You're on!`,
+        notNowTemplate: `You need five treats to play.`,
+        interact: () => {
+          const textText = treatGift > 0 ? 'Heads. You win!' : 'Tails. Tough luck.';
```

Comment:
This can be simplied - instead of adding textText to an object then setting text to it, we could just directly use this variable in setState()